File: mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(string prompt);
int main(void)
{
    //Call get_height function to get value
    int height = get_height("Height: ");
    //Make pyramid with value entered
    for (int row = 0; row < height + 1; row++)
    {
        //iterate over each row
        if (row > 0)
        {
            int sign = height - row;
            printf("%.*s", sign, "             ");
        }
        //iterate over each colum
        for (int colum = 0 ; colum < row ; colum++)
        {
            printf("#");
        }
        //Move to next line
        printf("\n");
    }
}



//Prompt user for value
int get_height(string prompt)
{
    int number;
    do
    {
        number = get_int("%s", prompt);
    }
    //Run loop only if number is between 1 and 8 inclusive
    while (number < 1 || number > 8);
    return number;
}

File: mario.py
from cs50 import get_int


def main():
    # Get correct height from use
    height = get_positive_int()
    # Make pyramid
    for row in range(height + 1):
        # iterate over each row
        if (row > 0):
            sign = height - row
            print(" " * sign, end="")
        # iterate over each colum
        for colum in range(row):
            print("#", end="")
        # Move to next line
        print()


def get_positive_int():
    while True:
        n = get_int("Height: ")
        if n >= 1 and n <= 8:
            break
    return n


main()
